index.html

```diff
@@ -1 +1,152 @@
-hello world
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Epilogue:wght@200;400;500;600;700&family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,700;1,200;1,400&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    </link>
+    <title>Soyeon Kwon's Resume</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div id="info">
+            <h1>Soyeon<br>Kwon</h1>
+            <h3>User Experience & Visual Designer</h3>
+            <ul>
+                <li>skwon02 (at) mica.edu</li>
+                <li>(443) 527-5810 </li>
+                <li>
+                    <a href="http://kwonsoyeon.com">kwonsoyeon.com</a>
+                </li>
+            </ul>
+        </div>
+        <div id="cotent-wrap">
+            <div class="column" id="et">
```

Comment:
I would challenge you to be more thoughtful about your naming, especially the IDs on your columns here. While you may know what `et` and `eas` stand for this week -- would you remember in a month or six? I always try to write IDs and class names in such a way that I am kind to my future self who may want to revisit or update this code :)